tox.ini

```diff
@@ -0,0 +1,49 @@
+[tox]
+skipsdist=True
+envlist = clean,ansible{2.12,2.13}-py{38,39,310}-{with_constraints,without_constraints},linters
+
+[testenv]
+description = Run the test-suite and generate a HTML coverage report
+deps =
+  pytest
+  pytest-cov
+  ansible2.12: ansible-core>2.12,<2.13
```

Comment:
We can drop the line and the 2 new ones. We don't need `ansible-core` to run our tests.